overcooked_ai_py/agents/zeroth_order_opt_active_states.py

```diff
@@ -0,0 +1,579 @@
+import time
```

Comment:
Again, I would move this file to your project repo, as it is currently quite specific to your project. For similar reasons to the agent.py file review (easier diff-viewing), I will do a more thorough review once it's in the PR on that repo.